### PR TITLE
new clean-up cluster policy

### PR DESCRIPTION
**Pull request by rcerven**
* CLOUDBLD-3769

Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -53,23 +53,45 @@ osbs_pruner_old_builds_command_build:
 - "--namespace={{ osbs_namespace }}"
 - builds
 - --orphans=true
-- --keep-younger-than=720h
```
Comment (MartinBasti) [inline defaults/main.yml:56]: It seems suspicious to me that this value is hardcoded in role instead of being configurable from playbook

Comment (rcerven) [reply defaults/main.yml:56]: configurable now

**Review by MartinBasti — APPROVED** [review]

File: defaults/main.yml [hunk 2]
```diff
@@ -35,41 +35,51 @@ osbs_buildroot_imagestream_live_tag: ''
...
-- prune
-- "--namespace={{ osbs_namespace }}"
-- builds
-- --orphans=true
-- --keep-younger-than=720h
-- --keep-complete=0
-- --keep-failed=0
-- --confirm
+osbs_pruner_builds_keep_complete: 1
+osbs_pruner_builds_keep_younger: '72h'
```
Comment (lkolacek) [inline defaults/main.yml:39]: @rcerven was this value change intentional? 720h against 72h

Comment (MartinBasti) [reply defaults/main.yml:39]: yes

Comment (lkolacek) [reply defaults/main.yml:39]: thanks, looked like possible typo and I wanted to be sure. See that second MR uses same constant. Looks good to me

**Review by lkolacek — APPROVED** [review]

**Pull request merged**